modules/mergeImageWithTelemetry/mergeImageWithTelemetryWorker.py

```diff
@@ -0,0 +1,45 @@
+from modules.mergeImageWithTelemetry.mergeImageWithTelemetry import MergeImageWithTelemetry
+
+def pipelineMergeWorker(pause, exitRequest, imagePipelineIn, telemetryPipelineIn, pipelineOut):
+    print("start pipeline merge")
+    mergeImageWithTelemetry = MergeImageWithTelemetry()
+    shouldGetImage = True
+    curImage = None
+
+    while True: 
+        if not exitRequest.empty():
+            return
+        
+        pause.acquire()
+        pause.release()
+        
+        if shouldGetImage:
```

Comment:
I think we should get the telemetry pipeline **first**, add it to the telemetryData array, then if an image isn't available, we still have the telemetry data. This means that, assuming the image pipeline producer runs slower than the telemetry pipeline producer, we should know that the item at the head of the telemetry pipeline (assuming LIFO) is newer than the item at the head of the image pipeline. Therefore, we know that the telemetryData with the closest timestamp to the image should be somewhere in the telemetryData array, not stuck in the telemetry pipeline.